src/platform/platform_posix.c

```diff
@@ -549,7 +555,19 @@ QuicProcCurrentNumber(
     void
     )
 {
+#if defined(QUIC_PLATFORM_LINUX)
     return (uint32_t)sched_getcpu();
+#elif defined(QUIC_PLATFORM_DARWIN)
+    int cpuinfo[4];   
+    asm("cpuid"
```

Comment:
I haven't found a better way to do this... :( 
This should probably get gated by `#if __x86_64__` for iOS.